daemon/algod/api/server/v2/handlers.go

```diff
@@ -467,6 +468,7 @@ func (v2 *Handlers) PendingTransactionInformation(ctx echo.Context, txid string,
 		response.ConfirmedRound = &r
 
 		response.ClosingAmount = &txn.ApplyData.ClosingAmount.Raw
+		response.AssetClosingAmount = &txn.ApplyData.AssetClosingAmount
```

Comment:
Yes, `Raw` is a member of `MicroAlgo`. Assets amount are `uint64` so we don't need it there.

Interesting, @tsachiherman and I talked about it but from a quick check, it seemed like it would remove it. Yes, if this is the case we will push a quick patch.